package.json

```diff
@@ -57,26 +57,23 @@
     "classnames": "^2.3.2",
     "constate": "^3.3.2",
     "date-fns": "^2.30.0",
+    "dayjs": "^1.11.10",
```

Comment:
denne trengs av familie-tidslinje. Ønsker å gjøre den til en dependecy og ikke peer dependency av familie-tidslinje så vi slipper å dra den inn her

Reply:
Aah, ja. Kanskje vi burde lage kort på å endre i familie-tidslinje til date-fns? 😁

Reply:
Yes! Men ønsker egentlig å gå bort fra familie-tidslinje på sikt, så usikker på om det er verdt det 🤔 har hvertfall laget denne for å se på muligheten for å bytte over: https://favro.com/organization/98c34fb974ce445eac854de0/1844bbac3b6605eacc8f5543?card=NAV-17941

Kan legge inn en kommentar der om at hvis man ser at det er umulig, så kan man lage oppgave på å bytte til date-fns